lib/publiccloud/instance.pm

```diff
@@ -47,7 +47,7 @@ C<<$instance->username()>>.
 Use argument C<rc_only> to only check for the return code of the command.
 =cut
 sub run_ssh_command {
-    my $self = shift;
+    my ($self) = @_;
     my %args = testapi::compat_args({cmd => undef}, ['cmd'], @_);
```

Comment:
```suggestion
    my ($self, @args) = @_;
    my %args = testapi::compat_args({cmd => undef}, ['cmd'], @args);
```

Reply:
this is not always wanted in my opinion. it would work for the most of the cases but i have code which cant apply this for really. if somehow ci checks for this reduce flexibility. the only way i am thinking it can fly is through code review where everyone one know that this is the main rule but it can accept a different implementation when this is required. but it needs everyone to share this policy equally.  just my opinion.

Reply:
@b10n1k well it's already in the contributing file, and this pr is stepping stone towards moving to having CI checks (And it's not just this, but many other things, so that we can get to a more robust codebase).

> this is not always wanted in my opinion.

Can you explain why?